city_scrapers/spiders/il_sex_offender_management.py

```diff
@@ -0,0 +1,234 @@
+            classification=BOARD,
+            start=start_time,
+            end=end_time,
+            all_day=False,
+            time_notes="",
+            location={
+                "address": "",
+                "name": "WebEx Meeting (See meeting links to confirm)",
+            },
+            links=self._make_links(None, start_time),
+            source=response.url,
+        )
+        meeting["status"] = self._get_status(meeting)
+        meeting["id"] = self._get_id(meeting)
+
+        yield meeting
+
+    def _video_start_end_time(self, clean_text_vid):
+        """Parse through JSON and return start, end naive datetime objects"""
+
+        # get date time formatted as '2020-09-16 13:00'
+        date_time = clean_text_vid["meetingData"]["startTime"][:-3]
+
+        # get meeting duration represented in minutes
+        scheduled_duration = clean_text_vid["meetingData"]["scheduledDuration"]
+
+        # create start and end time datetime naive objects
+        start_time = datetime.strptime("{}".format(date_time), "%Y-%m-%d %H:%M")
+
+        end_time = start_time + timedelta(minutes=scheduled_duration)
+
+        return start_time, end_time
+
+    def _parse_documents(self, response):
+        """Parse meeting information from agenda PDF"""
+
+        links_dict = {
+            "title": "Meeting Agenda",
+            "href": response.url,
+        }
+        clean_text = self._cleanUpPDF(response)
+
+        start_time = self._parse_start(clean_text)
+
+        meeting = Meeting(
+            title="MANAGEMENT BOARD",
+            description="",
+            classification=BOARD,
+            start=start_time,
+            end=self._parse_end(clean_text),
+            all_day=False,
+            time_notes="See agenda to confirm details",
+            location=self._parse_location(clean_text),
+            links=self._make_links(links_dict, start_time),
+            source=self.start_urls[0],
+        )
+        meeting["status"] = self._get_status(meeting, text=clean_text)
+        meeting["id"] = self._get_id(meeting)
+
+        yield meeting
+
+    def _make_links(self, agenda_dict, date_time_obj):
+        """Make list of agenda and minutes(if applicable)."""
+        links = []
+        if agenda_dict is not None:
+            links.append(agenda_dict)
+        for meeting in self.meeting_minutes:
+            if date_time_obj.date() == meeting[0].date():
+                # account for training meetings
+                if "training" in agenda_dict["href"].lower():
+                    if "training" in meeting[1].lower():
+                        minutes_dict = {"title": "Meeting Minutes", "href": meeting[1]}
+                        links.append(minutes_dict)
+                elif "training" not in meeting[1].lower():
+                    minutes_dict = {"title": "Meeting Minutes", "href": meeting[1]}
+                    links.append(minutes_dict)
+        return links
+
+    def _cleanUpPDF(self, response):
```

Comment:
Missed this on the first pass, but generally Python follows snake case vs camel case, so this would be "_clean_up_pdf". It's a convention thing, but following it consistently makes code easier to follow